### DISCUSSION TITLE
How to add nav_list in sidebar

### DISCUSSION THREAD

**Discussion by yuyulyu** [academicpages/academicpages.github.io#2266] [Q&A]
Hello everyone. I am trying to add a navigation list in sidebar, but I am not sure what is the correct structure to make it works.

Could someone please guide me on how to correctly set up the navigation list in the sidebar? Any help or pointers would be greatly appreciated!

Thank you in advance!

<img width="605" alt="image" src="https://github.com/user-attachments/assets/f7c8fe0c-138a-4f54-8bdb-58fe6180114f">

<img width="1218" alt="image" src="https://github.com/user-attachments/assets/7542b0d0-40a0-4c8c-97dc-c393eba2a781">

**Comment by rjzupkoii** [comment]
Do you mean move the top navigation menu to the side, or setup your own navigation on each page that replaces the existing author side bar?

**Reply by yuyulyu** [reply]
Yes, I mean the navigation on each page that could replace the author side bar. I have noticed that there is a ```page.sidebar.nav``` in <ins>includes/sidebar.html</ins>,  but I am not sure how to use it properly.

On the other hand, I also tried to use ```{% include toc %}``` instead, while seems like it doesn't  work well on my website too. It is not complete, and moves up with the content as I scroll up. Are there any existing solution can help with showing a sticky table of content for a page? 
<img width="299" alt="image" src="https://github.com/user-attachments/assets/75e51c56-fff0-40bd-8a48-4ac5ffaa7a42">

**Reply by rjzupkoii** [reply]
@yuyulyu Replacing the author side bar is possible and the `_includes/sidebar.html` is the place that you would need to make the modifications. You will need to get into a bit of how the templating code works, but as you can see from the code in `sidebar.html` it does adjust things a bit depending upon the page you are currently on, so you should be able to follow that pattern for your updates.
